pandas/core/generic.py

```diff
@@ -658,7 +658,7 @@ def size(self) -> int:
         """
         # error: Incompatible return value type (got "signedinteger[_64Bit]",
         # expected "int")  [return-value]
-        return np.prod(self.shape)  # type: ignore[return-value]
+        return int(np.prod(self.shape))  # type: ignore[return-value]
```

Comment:
Now it should be possible to remove the error&ignore comment, or?

Reply:
Just a reminder on this comment - you should be able to remove the `type: ignore` comment (and the comment above this line)